content/cli/overview.md

```diff
@@ -1,23 +1,79 @@
+
+The Nest CLI is based on the [Angular Devkit](https://github.com/angular/devkit) package, and works for server-side applications in much the same way that the [Angular CLI](https://angular.io/cli) assists development of client-side Angular apps.
 
 #### Installation
 
-Install the CLI using **NPM**:
+Install the CLI using **npm**.
+
+> info **Warning** you should use the `-g` option to install `@nestjs/cli` globally, as some of its functions operate outside of the context of a `package.json` file.
 
 ```bash
 $ npm install -g @nestjs/cli
 ```
 
 #### Basic workflow
 
-Once the installation process is finished, you should be able to invoke CLI commands directly from your command line through the `nest` executable. In order to get a list of available functionalities, use the following command:
+Once installed, you can invoke CLI commands directly from your OS command line through the `nest` executable.  Show the available `nest` commands by entering the following:
 
 ```bash
 $ nest --help
 ```
 
-#### Contribute
+Get help on an individual command using the following construct. Substitute any command, like `new`, `add`, etc., where you see `generate` in the example below to get detailed help on that command:
+
+```bash
+$ nest generate --help
+```
+
+To create, build and run a new basic Nest project in development mode, go to the folder that should be the parent of your new project, and run the following commands:
+
+```bash
+$ nest new my-nest-project
+$ cd my-nest-project
+$ nest start --watch
+```
+
+In your browser, open [http://localhost:3000](http://localhost:3000) to see the new application running. With the `--watch` flag, the app will automatically recompile and reload when you change any of the source files.
+
+#### Workspaces
+
+When you run `nest new`, Nest generates a boilerplate application structure by creating a new folder and populating an initial set of files.  You can continue working in this default structure, adding new components, as described throughout this documentation.  We refer to this as **single project mode**.  Nest also supports the notion of a multi-project **workspace mode**.  You can read the section on [Workspaces](/cli/workspaces) to decide which mode is most suitable for you.  Aside from a very few specific considerations around how the **build** process works (essentially, workspaces automate the build complexities that can sometimes arise from monorepo-style project structures), the rest of this documentation applies equally to both single project and workspace mode approaches.  In fact, you can easily switch from single project mode to workspace mode at any time in the future, so you can safely defer this decision while you're still learning about Nest.
```

Comment:
We should add a note here that if you want to simply develop multiple applications that would never share anything/separate libraries that will be eventually published to NPM - multiple workspaces can be used (separate dependencies `node_modules`, separate package.json etc).

Reply:
Next update adds details on this.  Will need review to judge if it resolves this.